pkgs/top-level/python-packages.nix

```diff
@@ -18304,6 +18304,21 @@ EOF
   h11 = callPackage ../development/python-modules/h11 { };
 
   python-docx = callPackage ../development/python-modules/python-docx { };
+
+  yeelight = buildPythonPackage rec {
+    name = "yeelight-${version}";
```

Comment:
Please specify `pname = "yeelight"` instead. `name` will automatically be set by `buildPythonPackage`.